src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -439,10 +439,42 @@ public void initSendable(SendableBuilder builder) {
     SendableHelper.addChild(builder, this, armController, "ArmController");
     SendableHelper.addChild(builder, this, elevatorController, "ElevatorController");
     builder.addDoubleProperty(
-        "Elevator Position", elevatorLeftEncoder::getPosition, elevatorLeftEncoder::setPosition);
-    builder.addDoubleProperty("Arm Position", armEncoder::getPosition, armEncoder::setPosition);
-    builder.addBooleanProperty("Done Scoring", this::doneScoring, null);
+        "Elevator Position (in)",
+        elevatorLeftEncoder::getPosition,
+        elevatorLeftEncoder::setPosition);
+    builder.addDoubleProperty(
+        "Elevator Vel (in)", elevatorLeftEncoder::getVelocity, elevatorLeftEncoder::setPosition);
```

Comment:
Why is the setter `setPosition`? I don't think there should be a setter here.

Reply:
Whoops copy paste error removed